specials/SPC18_Matias_SpiesVsMercs.Altis/victory_handler.sqf

```diff
@@ -5,13 +5,22 @@ if (spies_extracted == true) then
 	[mission_victoryHandlerPFH] call CBA_fnc_removePerFrameHandler;
 };
 
-//This will count the alive players
-_alivePlayers = {isPlayer _x && alive _x && side _x == west} count allUnits;
+//This will count the alive spies
+_aliveSpies = {isPlayer _x && alive _x && side _x == west} count allUnits;
 
 //Mercs win end
-if (_alivePlayers == 0) then
+if (_aliveSpies == 0) then
 {
```

Comment:
Need to move `{` to the same line as `then`, this is how the majority of standard style guides treat blocks
```sqf
if (_aliveSpies == 0) then {
```